### PR TITLE
Uncoherent Error Message #23

### PR DESCRIPTION
**Pull request by giulianogiari**
fixed the error message as detailed in #23

### CODE REVIEW

File: src/core/cpuCO2.py [hunk 1]
```diff
@@ -23,7 +23,11 @@ def cpu_co2(country_code = None) -> None:
         lat = geolocation['latitude']
 
         # Retrieve data from Co2Signal API
-        co2data = api_calls.get_request_co2signal(lon, lat)
+        try: 
+            co2data = api_calls.get_request_co2signal(lon=lon, lat=lat)
+        except Exception as e:
+            print("\nco2data for your location could not be retrieved."
+                "\nPlease try again later or specify a country code")
```
Comment (FrnRmn) [inline src/core/cpuCO2.py]: I correctly see the error message in I insert an error at line 27
image
However, the script is not stopping and goes on, producing a code crash or a run with wrong data. 
I would suggest to force the stop of the application after the error message.

File: src/core/cpuCO2.py [hunk 2]
```diff
@@ -56,10 +60,12 @@ def cpu_co2(country_code = None) -> None:
             sys.exit(app.exec_())
         except SystemExit:
             print('Closing Window...')
-
-    except KeyError:
-        print('Sorry, your country is not present in the Electricity Map ' +
-              'database, it is therefore not possible to retrieve CO2 emissions data.')
+    except Exception as e:
```
Comment (FrnRmn) [inline src/core/cpuCO2.py:79]: This looks good to me, nice Idea to add the repository ref!

File: src/core/cpuCO2.py [hunk 3]
```diff
@@ -19,51 +20,74 @@ def cpu_co2(country_code = None) -> None:
...
-        co2data = api_calls.get_request_co2signal(lon, lat)
-        country_code = co2data['countryCode']
+        try:
+            co2data = api_calls.get_request_co2signal(lon=lon, lat=lat)
+        except Exception as e:
+            print(
+                "\nco2data for your location could not be retrieved."
+                "\nPlease try again later or specify a country code"
+            )
+            os._exit(1)
```
Comment (FrnRmn) [inline src/core/cpuCO2.py]: This is the only row I added from your PR, all the rest is the Black formatting

**Review by FrnRmn — APPROVED** [review]
Can be merged for me if everything looks ok @giulianogiari @MattiaSilvestri

File: src/core/cpuCO2.py (hunk 2 above)
Comment (MattiaSilvestri) [reply src/core/cpuCO2.py:79]: Yes nice idea.

**Review by MattiaSilvestri — APPROVED** [review]

**Pull request merged**